src/grafana_dashboards/zookeeper-metrics.json

```diff
@@ -8967,36 +8967,14 @@
         "allValue": null,
         "current": {},
         "datasource": "${prometheusds}",
-        "definition": "label_values(job)",
-        "hide": 0,
-        "includeAll": false,
-        "label": "Cluster",
-        "multi": false,
-        "name": "job",
-        "options": [],
-        "query": "label_values(job)",
-        "refresh": 1,
-        "regex": "",
-        "skipUrlSync": false,
-        "sort": 1,
-        "tagValuesQuery": "",
-        "tags": [],
-        "tagsQuery": "",
-        "type": "query",
-        "useTags": false
-      },
-      {
-        "allValue": null,
-        "current": {},
-        "datasource": "${prometheusds}",
-        "definition": "label_values({job=~\"$job\"}, instance)",
+        "definition": "label_values(juju_unit)",
```

Comment:
**todo** Uhm, why not skipping this all together? the `juju_unit` will anyway be injected by cos, also by subselecting only relevant labels (e.g. if you select `juju_application==kafka` then you would only get the `juju_unit` of kakfa. As it is right now, I believe you would always get all units, also if they are not relevant.

Reply:
Done in bf39154.
Thanks for reminding me, I completely skipped over this in your branch